game_eggs/tycoon_games/openrct2/egg-openrct2.json

```
@@ -0,0 +1,193 @@
+            "container": "debian:bullseye-slim",
+            "entrypoint": "\/bin\/bash"
+        }
+    },
+    "variables": [
+        {
+            "name": "Save File",
+            "description": "Save file (.sv4 \/ .sv6 \/ .park) to load. Can be a file path or can be a webhosted file URL. An existing save file must be specified; the server cannot generate a new scenario\/save from scratch.",
+            "env_variable": "SAVE_URI",
+            "default_value": "ServerData\/save\/save.park",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string|min:5"
```

```suggestion
            "rules": "required|string|min:4"
```
`.sv4` is a valid filename

---

Thank you for catching this; think I was tired when I switched the default save file from `.sv6` to `.park`.